PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -210,8 +210,22 @@ function New-Report {
 
     # Handle AAD-specific reporting
     if ($BaselineName -eq "aad") {
+        
+        # Load the CSV file
+        $csvData = Import-Csv -Path "Product Names for Licensing.csv"
+        
         $LicenseInfoArray = $SettingsExport.license_information | ForEach-Object {
+
+            $SkuID = $_.SkuId
+            # Find the corresponding product name
+            $matchingRow = $csvData | Where-Object { $_.GUID -eq $SkuID } | Select-Object -First 1
+            $productName = ""
+            if ($matchingRow) {
+                $productName = $matchingRow.'Product_Display_Name'
```

Comment:
Forgive my powershell ignorance, but does this properly select from the matching row object?

Would it be better to use camelCase headers in the CSV or is the CSF just a direct import from Microsoft's website?